src/telemetry/telemetryController.ts

```diff
@@ -34,14 +34,15 @@ type CloudInfo = {
 
 type LinkClickedTelemetryEventProperties = {
   screen: string;
-  link_id: string;
+  link_id: string; // eslint-disable-line camelcase
```

Comment:
Probably we can disable it for the whole file. Because we use camelcase props for telemetry events only in this file. And later we can have more such events.